app/models/sales_log.rb

```diff
@@ -61,6 +61,10 @@ def self.editable_fields
     attribute_names
   end
 
```

Comment:
This felt redundant to implement as though I should have access to the underlying log in the view however It is being done for other fields, I'm guessing the log itself is a database object and we're working with an object representation of it instead of the raw data? I'd be interested in the distinction between the two.

Reply:
this does look redundant. I would expect that wherever you are calling sales_log.purchaser_code you would be able to call sales_log.purchid

In this case I'm personally fine with it anyway since purchid is meaningless to humans